src/Explorer/Explorer.tsx

```diff
@@ -370,8 +370,11 @@ export default class Explorer {
         });
         useNotebook.getState().setIsAllocating(true);
         connectionInfo = await this.phoenixClient.allocateContainer(provisionData);
-        if (!connectionInfo?.data?.notebookServerUrl) {
-          throw new Error(`NotebookServerUrl is invalid!`);
+        if (connectionInfo.status !== HttpStatusCodes.OK) {
```

Comment:
this check is not needed as it is handled in the phoenix client. This wont be hit